swapserverrpc/Dockerfile

```diff
@@ -1,21 +1,25 @@
-FROM golang:1.16.3-buster
+FROM golang:1.21.9-bookworm
```

Comment:
Reminder to push a new tag for the `swapserverrpc` module after merging this PR.